build-system/pr-check/single-pass-tests.js

```diff
@@ -27,6 +27,7 @@ const {
   startTimer,
   stopTimer,
   timedExecOrDie: timedExecOrDieBase,
+  timedExecOrDieWithDnsMonitor: timedExecOrDieWithDnsMonitorBase,
```

Comment:
With my suggestion to change how `gulp dns-monitor` works, this file can be reverted.